hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieTableServiceClient.java

```diff
@@ -558,9 +558,6 @@ protected void runTableServicesInline(HoodieTable table, HoodieCommitMetadata me
       return;
     }
 
-    if (config.isMetadataTableEnabled()) {
```

Comment:
I remember there’s a reason to sync the FS view for MDT.  Is this unnecessary now?

Reply:
I have checked all the inline table services, all we need is the timeline, there is no need to sync the view because each service actually re-create their own table when necessary.